prow/plugins/trickortreat/trickortreat.go

```diff
@@ -0,0 +1,190 @@
+		"https://c4.wallpaperflare.com/wallpaper/344/140/1013/chocolate-box-candies-allsorts-wallpaper-preview.jpg",
+		"https://images.pexels.com/photos/3440657/pexels-photo-3440657.jpeg?cs=srgb&dl=photo-of-jar-with-candies-near-christmas-balls-3440657.jpg&fm=jpg",
+		"https://images.pexels.com/photos/4114979/pexels-photo-4114979.jpeg?auto=compress&cs=tinysrgb&fit=crop&h=627&w=1200",
+		"https://cdn.pixabay.com/photo/2017/07/01/22/30/cartoon-2462970_960_720.png",
+		"https://img.rawpixel.com/s3fs-private/rawpixel_images/website_content/a019-jakubk-0900-sweet-candies-store3.jpg?auto=format&bg=transparent&con=3&cs=srgb&dpr=1&fm=jpg&ixlib=php-3.1.0&mark=rawpixel-watermark.png&markalpha=90&markpad=13&markscale=10&markx=25&q=75&usm=15&vib=3&w=1400&s=625c13af905d7c79413535c86b0249ff",
+		"https://cdn.pixabay.com/photo/2013/07/12/11/59/candies-145068_640.png",
+		"https://images.pexels.com/photos/4016594/pexels-photo-4016594.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+		"https://i1.pickpik.com/photos/128/614/371/candies-colorful-store-sweet-thumb.jpg",
+		"https://cdn.pixabay.com/photo/2016/07/13/09/43/gummy-bears-1514016_640.png",
+		"https://images.pexels.com/photos/2064126/pexels-photo-2064126.jpeg?auto=compress&cs=tinysrgb&fit=crop&h=627&w=1200",
+		"https://images.pexels.com/photos/2559743/pexels-photo-2559743.jpeg?cs=srgb&dl=awesome-candies-dessert-milkshake-2559743.jpg&fm=jpg",
+		"https://images.pexels.com/photos/4114980/pexels-photo-4114980.jpeg?cs=srgb&dl=candies-and-flowers-on-black-surface-4114980.jpg&fm=jpg",
+		"https://get.pxhere.com/photo/play-sweet-food-color-child-dessert-colour-children-background-bonbons-candy-sweetness-smarties-yummy-sprinkles-confectionery-lentils-snack-food-jelly-bean-1360826.jpg",
+		"https://images.pexels.com/photos/413610/pexels-photo-413610.jpeg?cs=srgb&dl=candies-candy-candy-cane-child-413610.jpg&fm=jpg",
+		"https://get.pxhere.com/photo/group-food-green-red-brown-colorful-yellow-snack-dessert-toy-sugar-bright-multicolored-candy-confection-skittles-assortment-confectionery-candies-jelly-bean-742963.jpg",
+		"https://images.pexels.com/photos/1375807/pexels-photo-1375807.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+		"https://images.pexels.com/photos/1578293/pexels-photo-1578293.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+		"https://images.pexels.com/photos/307280/pexels-photo-307280.jpeg?cs=srgb&dl=cakes-candies-food-sweets-307280.jpg&fm=jpg",
+		"https://images.pexels.com/photos/4015264/pexels-photo-4015264.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+	}
+)
+
+const (
+	pluginName = "trickortreat"
+)
+
+func init() {
+	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, _ []config.OrgRepo) (*pluginhelp.PluginHelp, error) {
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: "The trickortreat plugin adds a trickortreat image to an issue or PR in response to the `/trick-or-treat` command.",
```

Comment:
nit: Should this also use 'candy' like below?

Reply:
Good catch, fixed